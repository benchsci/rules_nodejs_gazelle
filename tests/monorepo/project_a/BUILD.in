# gazelle:js_package_file package.json :node_modules
# gazelle:js_collect_all