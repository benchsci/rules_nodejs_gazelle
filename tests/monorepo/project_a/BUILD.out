load("@aspect_rules_js//js:defs.bzl", "js_library")
load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:js_package_file package.json :node_modules
# gazelle:js_collect_all

js_library(
    name = "package_json",
    srcs = ["package.json"],
)

ts_project(
    name = "project_a",
    srcs = ["a.ts"],
    data = ["//project_a:node_modules/@emotion/react"],
    deps = ["//project_a:node_modules/@emotion/react"],
)